components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -329,18 +329,22 @@ rendering:
     ...
 ```
 
-Now you can use the timespan from the application's code:
+Now you can use the timespan from the application's code. Each time interval that
+the timespan metric records must be associated with an object provided by the user.
+This is so that intervals can be measured concurrently.  For example, to measure page
+load time on a number of tabs that are loading at the same time, each time interval
+should be associated with an object that uniquely represents each tab.
 
 ```Kotlin
-import org.mozilla.yourApplication.GleanMetrics.Rendering
+import org.mozilla.yourApplication.GleanMetrics.Pages
+
+fun onPageStart(e: Event) {
+    Pages.pageLoad.start(e.target)
```

Comment:
Do we have to require the parameter even for timespans?
I can see concurrency issues with the timing distributions, but i wonder if we can avoid the parameter for the timespan API.

Reply:
I don't think there's a requirement that timespans are used once per application run, since they sum together a bunch of timespans.  It could be used to measure the total time spent performing a task, which might be performed on multiple threads, even concurrently.  I think it's safest (fewest foot-guns) to require the parameter, since